### PR TITLE
Prepare release 0.65.0

### PR DESCRIPTION
**Pull request by avishniakov**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [x] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

**Comment by avishniakov** [comment]
@AlexejPenner I didn't push Notebook changes just yet, to let CI finish

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,77 @@
 <!-- markdown-link-check-disable -->
 
+# 0.65.0
+
+## New Features and Improvements
+
+### Lightning AI Studio Orchestrator
```
Comment (schustmi) [inline RELEASE_NOTES.md:7]: This should not be in here IMO, this should not be an official feature until the orchestrator is stable. And it's far from that for now.

**Review by schustmi — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (avishniakov) [reply RELEASE_NOTES.md:7]: Ok, I will silent it down

**Comment by avishniakov** [comment]
CI passed, pushing cosmetic changes:
- https://github.com/zenml-io/zenml/actions/runs/10598332452/job/29370663838?pr=2960
- https://github.com/zenml-io/zenml/actions/runs/10598332449/job/29371689285?pr=2960

**Review by schustmi — APPROVED** [review]

**Pull request merged**